<launch>
  <rosparam command="load" file="$(find perception_pc_segmentation)/config/feature_seg_camera_topics.yaml"/>
  <rosparam command="load" file="$(find perception_pc_segmentation)/config/parameters.yaml"/>

  <param name="feature_segmentation/use_compressed_streams" value="false" />
  <param name="feature_segmentation/debug_mode" value="false" />
  <param name="feature_segmentation/model_path" value="$(find apc_2016_mesh_models)/object_models"/>

  <node name="feature_segmentation" pkg="perception_pc_segmentation" type="feature_segmentation" output="log" respawn="true"/>

</launch>
